### PR TITLE
Ts dataset

### PR DESCRIPTION
**Pull request by aa25desh**
At moment `gunpoint` & `Adiac` are working with `ts_dataset`.
But for some reason on `ACSF1` dataset `ts_dataset` just quits Julia.

### CODE REVIEW

File: src/datapath.jl [hunk 1]
```diff
@@ -80,6 +89,54 @@ function TSdataset(filepath::String)
...
+"""
+   `load_ts_file(fpath)`
+`load_ts_file` takes the path to `ts` files and returns the table (IndexedTable)
+and Array, where `fpath` is path to `ts` file located on your computer.
+eg
+```julia
+   table_a, arr = load_ts_file("/Users/your_user_name/../GunPoint_TRAIN.ts")
+```
+"""
+function load_ts_file(fpath)
```
Comment (mloning) [inline src/datapath.jl]: add boolean kwarg argument `return_array=False` to either return array or table

File: src/datapath.jl [hunk 2]
```diff
@@ -80,6 +89,54 @@ function TSdataset(filepath::String)
...
+         return load_ts_file(fpath)
+    elseif split == nothing
+         v = []
+         for i in ["test", "train"]
+             fpath = joinpath(DATA_DIR, dataset, string(dataset, "_", uppercase(i), ".ts" ))
+             push!(v, fpath)
+         end
+         test, train =  load_ts_file.(v)
+         return merge(test[1], train[1]), merge(test[2], train[2])
+    end
```
Comment (mloning) [inline src/datapath.jl:146]: add else clause which raises error if split is not one of nothing, train, test

File: src/datapath.jl (hunk 1 above)
Comment (aa25desh) [reply src/datapath.jl]: I have added kwarg argument.

File: src/datapath.jl (hunk 2 above)
Comment (aa25desh) [reply src/datapath.jl:146]: Added the error massage.

**Comment by mloning** [comment]
Looks good to me!

**Pull request merged**